src/main/scala/it/pagopa/interop/backendforfrontend/service/impl/TenantProcessServiceImpl.scala

```diff
@@ -53,14 +53,14 @@ class TenantProcessServiceImpl(tenantprocessUrl: String, blockingEc: ExecutionCo
       invoker.invoke(request, s"Revoking declared attribute $attributeId to requester Tenant")
     }
 
-  def selfcareUpsertTenant(origin: String, externalId: String, name: String)(
+  def selfcareUpsertTenant(origin: String, externalId: String, description: String)(
```

Comment:
isn't this still the name?
```suggestion
  def selfcareUpsertTenant(origin: String, externalId: String, name: String)(
```